aca_preview/preview.py

```diff
@@ -167,18 +192,90 @@ def get_summary_text(acas):
 
 class ACAReviewTable(ACATable):
     @classmethod
-    def add_review_methods(cls, aca):
+    def add_review_methods(cls, aca, *, obsid=None, loud=False, preview_dir='.'):
         """Add review methods to ``aca`` object *in-place*.
 
         - Change ``aca.__class__`` to ``cls``
         - Add ``context`` and ``messages`` properties.
 
         :param aca: ACATable object, modified in place.
+        :param obsid: obsid (optional)
+        :param loud: print processing status info (default=False)
 
         """
         aca.__class__ = cls
         aca.context = {}  # Jinja2 context for output HTML review
         aca.messages = []  # Warning messages
+        aca.loud = loud
+
+        # Input obsid could be a string repr of a number that might have have
+        # up to 1 decimal point.  This is the case when obsid is taken from the
+        # ORviewer dict of ACATable pickles from prelim review.  Tidy things up
+        # in these cases.
+        if obsid is not None:
+            f_obsid = round(float(obsid), 1)
+            i_obsid = int(obsid)
```

Comment:
```
ine 217, in add_review_methods
    i_obsid = int(obsid)
ValueError: invalid literal for int() with base 10: '20208.1'
```
Also, what's the best way to test entry-point scripts?  I'm used to just having a __name__ == main in the source script to run it from the source dir.

Reply:
I'm guessing you just intended
```
i_obsid = int(f_obsid)
```